pom.xml

```diff
@@ -86,6 +86,13 @@
                 </includes>
                 <filtering>false</filtering>
             </resource>
+            <resource>
```

Comment:
Is this PR a fix for https://github.com/telefonicaid/perseo-core/issues/19?

Maybe should it be mentioned in the CHANGES_NEXT_RELEASE file?